src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb

```diff
@@ -14,7 +14,7 @@ class Cluster
       CLUSTER_VM_HOST_RULE_SHOULD = 'SHOULD'.freeze
       CLUSTER_VM_HOST_RULE_MUST = 'MUST'.freeze
       PROPERTIES = %w(name datastore resourcePool host)
-      HOST_PROPERTIES = %w(name hardware.memorySize runtime.connectionState runtime.inMaintenanceMode runtime.powerState)
+      HOST_PROPERTIES = %w(name hardware.memorySize runtime.connectionState runtime.inMaintenanceMode runtime.powerState summary.hardware.cpuMhz)
```

Comment:
HOST_PROPERTIES doesn't seem like it's used anymore and can be deleted rather than updated